sbncode/TPCReco/VertexStub/VertexStubTracker_module.cc

```diff
@@ -114,8 +118,9 @@ void sbn::VertexStubTracker::produce(art::Event& e)
   auto const clock_data = art::ServiceHandle<detinfo::DetectorClocksService const>()->DataFor(e);
   auto const dprop =
     art::ServiceHandle<detinfo::DetectorPropertiesService const>()->DataFor(e, clock_data);
-  // TODO: fix -- for now, use a null space-charge service
-  const spacecharge::SpaceCharge *sce = nullptr;
+  const spacecharge::SpaceCharge *sce = lar::providerFrom<spacecharge::SpaceChargeService>();
+  // If we are not correcting for SCE, then blank out the service
+  if (!fCorrectSCE) sce = nullptr;
```

Comment:
Same comment and _recommendation_ as before:
```suggestion
  // If we are not correcting for SCE, then blank out the service
  const spacecharge::SpaceCharge *sce = fCorrectSCE? lar::providerFrom<spacecharge::SpaceChargeService>(): nullptr;
```